### DISCUSSION TITLE
2022 Hacktember to Remember followups

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcamap-react#52] [General]
Hey @envm92 and @joyliao07,

Have you two discussed or decided upon the best way to share your code improvements?

Let me know if you need help submitting a pull request (via a feature branch or fork of `orcamap-react`, or if you need different access roles for the repo.

Here are links to the [planning document](https://docs.google.com/document/d/1Undj6mwUR2eHh-qO2Vk27O7mui4_VTIZbHMkolJ2H3Y/edit#) we used and the [final slide deck](https://docs.google.com/presentation/d/10s_mY7LmPnn3lckYNFJC2zJdD_1_xH8fF24eF0CK2kk/edit#slide=id.g152b1b0a06f_0_6). It's so exciting that you were able to get some recent marine mammal sightings on the open layers map!

<img width="1049" alt="Screen Shot 2022-09-15 at 6 52 02 AM" src="https://user-images.githubusercontent.com/14044595/190422156-85d2ba15-d199-4c78-8ba5-3b47021c782b.png">

Thanks again for hacking with Orcasound last Saturday,
Scott

**Comment by joyliao07** [comment]
Hi Scott,


Yes, it was a good event! Here is the follow-up from me:
1. I still need permission to push code to the "orcamap-react" repository
(Right now I only have access to "orcamap")
2. The UI changes that you captured in the screenshot exist only on Erika's
laptop. Before we departed, I believed that she said she could be reached
via Slack.


Joyce Liao
Software Developer
425.269.8065

**Comment by envm92** [comment]
Hi!

Sorry for the late reply,
 I'm having an issue uploading my code. I got this message
``remote: Permission to orcasound/orcamap-react.git denied to envm92.``
I follow the instruction on
https://drive.google.com/file/d/1GHuylE3qPm2UAyUA_f1yOtMgutBpXF6V/view ,
but I didn't have any luck.

I'm going to do a fork and then open a PR to share the code

Thanks

**Comment by envm92** [comment]
Hi I was able to upload my code https://github.com/envm92/orcamap-react/tree/dev/hackaton-09-2022/eri , I wonder if I could propose a web element to create this dynamic map, that also could embebed at any web page. I don't know how I could move this idea.

**Comment by joyliao07** [comment]
Erika, It sounds like a great idea. Please keep me in the loop.

Ps - I also had the react-mapbox library working. However, it involves
using a public token from Mapbox. I want to make sure that there won't be
any security issues if I push the code up.


Joyce

**Comment by ayushman-t** [comment]
Hello @joyliao07, Can you please share the code you authored during the 2022 hackathon? I'm in the process of advancing the project and have already obtained the changes @envm92 made from her local repository.

If possible, could both of you create a pull request so that code changes can be attributed to the respective developers? Thanks

**Reply by joyliao07** [reply]
I had to change my laptops. I am afraid I no longer have it.

**Reply by ayushman-t** [reply]
No problem.
